### PR TITLE
Add installation notes, and move code to src folder

### PR DESCRIPTION
**Pull request by amitaibu**
#2
- Added installation notes
- Moved code under `src` as this seems to be the best practice ([here's](https://github.com/phadej/github) an example)

### CODE REVIEW

**Comment by amitaibu** [comment]
I saw the changes worked, but haven't wired it into my own app yet, so it probably deserves some more testing ;)

**Comment by axel-angel** [comment]
Can you also test the forked repo, it seems to improve on some aspects like stack integration. Keep me posted about your testing, I'll merge when you are confident about it.

**Comment by axel-angel** [comment]
Thanks for your help.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+As this package is not published in Hackage, you should install it locally.
+
+1. From the root of your project:
+
+```bash
+# Clone the repository
+git clone https://github.com/axel-angel/yesod-auth-token.git
+
+# Delete the package's own .git folder
+rm -rf yesod-auth-token/.git
```
Comment (axel-angel) [inline README.md:12]: Why?

**Comment by ghost** [comment]
@amitaibu You should install it as a submodule of your own project using git submodule add ... and not clone it and remove the VCS... Also if ever I added a stack config file, so you really only should need to clone it and stack will find out about the package by itself. I'll have a look later in the day at your PR (I'm kinda taking over from @axel-angel ;))

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md:12]: You should use git submodules to do that...

**Comment by ghost** [comment]
All right, I merged your PR (kept some version constraints though, but updated them so they match stack's LTS-6.6. I'll also update the README so it uses git submodules instead of un-VCS-ing the code (which is Bad)

**Comment by axel-angel** [comment]
Thanks :+1:

**Comment by amitaibu** [comment]
:+1:

**Comment by ghost** [comment]
my pleasure :wink:

**Pull request merged**